src/components/base/GlossaryDefinitions.vue

```diff
@@ -0,0 +1,8 @@
+<style lang="scss" scoped>
+@import "@/assets/styles/main.scss";
+.glossary-definition {
+  font-size: is-size-7;
+  padding-left: px-4;
+  padding-top: py-1;
+}
+</style>
```

Comment:
```suggestion
<template>
  <p class="py-1 is-size-7">
    <strong>{{term}}:</strong>
    <slot />
  </p>
</template>

<script lang="ts" setup>
defineProps<{
  term: string
}>()
</script>
```

This is what I had in mind for the component